django-server/server/api/views.py

```diff
@@ -197,13 +192,14 @@ def get_queryset(self):
             return []
         cafeteria_pk = self.kwargs.get('cafeteria_pk')
         req_count = self.request.query_params.get('count')
-        start_string = self.request.query_params.get('timestamp_start')
-        end_string = self.request.query_params.get('timestamp_end')
-        divider = self.divider
+        start_string = self.request.query_params.get('start_timestamp')
+        end_string = self.request.query_params.get('end_timestamp')
+        group_by_string = self.request.query_params.get('group_by')
+        divider = get_divider(group_by_string)()
 
-        timestamp_start = datetime.min + divider.get_timestamp_delta(divider, datetime.min) \
+        timestamp_start = datetime.min + divider.get_timestamp_delta(datetime.min) \
             if start_string is None else timestamp_parse(start_string)
```

Comment:
These inline conditional statements add complexity to code. Try not to use them in future.

Reply:
You can run `flake8-cognitive-complexity` locally to test code for such structures.

Reply:
wil fix